### ISSUE TITLE
That not working it need root access

### ISSUE THREAD

**Issue by Vision200617380** [TermuxHackz/wifi-hacker#6]
![Screenshot_20210930-110614_Termux](https://user-images.githubusercontent.com/91659597/135411704-85c4e37f-2d0e-4132-8080-21a07b940343.jpg)
That required boot access I need that

**Comment by TermuxHackz** [comment]
Yes it needs root access, note that termux cannot hack WiFi networks, it
has to be rooted or you must have a rooted mobile device
Thank you.

**Comment by Kdgdj** [comment]
One problem
